### DISCUSSION TITLE
GDG Road to Certification meetups

### DISCUSSION THREAD

**Discussion by jlandure** [lyonjs/shortvid.io#219] [Conference / Meetup Communication]
We are launching a Road to Certification : a six week initiative to learn and go to the Professional Cloud Architect examination.

😇  And I would like some videos thanks to your work! 💚 

We are doing a "Session Info" this Thursday : https://gdg.community.dev/events/details/google-gdg-cloud-nantes-presents-certification-study-group-info-session-session-dinformation/
And then, we are doing "🎓 Certification Study Group - Road to Certification : PCA 🇫🇷" every week the Tuesday noon
see listing here: https://gdg.community.dev/gdg-cloud-nantes/

Here are some materials:
- Logo: https://drive.google.com/file/d/1MnfXX2ppk8tZZ1uI16BF7MoQ0udMl8sv/view?usp=share_link
- small icon: https://drive.google.com/file/d/1ljM68mm-jRvb-wLiCgyEAqL-2-jCs7bZ/view?usp=share_link
- Event banner: https://drive.google.com/file/d/1D-d5zxGDfZGdnNbLW7clNMCHKlcWZ8ht/view?usp=share_link

Already **90 registrations!** 🎉 

List of GDG participating: 

Hors-France GDGs:
- :white_check_mark:  GDG Cloud Abidjan :flag-ci:
- GDG Mons :flag-be:  
France GDGs: :fr:
- :white_check_mark:  GDG Nantes / GDG Cloud Nantes
- GDG Lille 
- GDG Tours 
- :white_check_mark:  GDG Le Mans
- :white_check_mark:  GDG Cloud Paris
- :white_check_mark:  GDG Paris
- GDG Bordeaux 
- :white_check_mark:  GDG Cloud Lyon
GDG Sophia antipolis 

ping @CruuzAzul

**Comment by mickaelalvs** [comment]
We'll see what we can do about that 😎

**Reply by mickaelalvs** [reply]
@jlandure I have two small questions : 
- Can you tell us what information you would like to put on the videos (dates, ...)
- Do you have the assets in svg ? It's for the animations, otherwise we'll do without 😊

**Comment by jlandure** [comment]
Yes 🙌 
4 new assets with one in SVG
- [Vector] Road to Google Developers Certification - Google Cloud.svg : https://drive.google.com/file/d/1LHw7JdpBh7f_Eu_N3GgBWIFisr1vpFas/view?usp=share_link
- Homepage: https://drive.google.com/file/d/1mLU9jb2j-fXC0WdG5iaIcxPqV7FS-gxe/view?usp=share_link
- Social gif 1 : https://drive.google.com/file/d/1VL3CXSFhI5e6f6nlJ99cE5iABRCrwTaZ/view?usp=share_link
- Social gif 2 : https://drive.google.com/file/d/19Y0NRm1ulUrAXADflWRx9eJyLJa9uGsG/view?usp=share_link

I hope this will be useful ! 💚

**Comment by jlandure** [comment]
For the video, I would like an intro video with general info on the Road to Certification.

And then, a video on our event this thursday : https://gdg.community.dev/events/details/google-gdg-cloud-nantes-presents-certification-study-group-info-session-session-dinformation/ organized with many GDGs

See also my tweet: https://twitter.com/jlandure/status/1640634021418762242

**Reply by mickaelalvs** [reply]
Thanks ! 🙏🏼
For the video of Thursday it might be complicated but we can make a small video with an intro and the announcement of the next events 😉

**Reply by jlandure** [reply]
Ok perfect 💚

**Comment by jlandure** [comment]
Any news? We already are in the Week2 of the program. 😊

**Reply by mickaelalvs** [reply]
Hey sorry @jlandure 😪 I missed this post... 😭 We created a template : 


https://github.com/lyonjs/shortvid.io/assets/60877626/effdef51-591a-468c-8934-551f1007683d

ℹ️ This is a demo, you can change the background colour, the texts ...

Normally we can fill it with whatever you need and we should be able to generate the videos ! (We'll be able to generate them from the site soon 🤞🏼)

Is it still necessary ? We can make this available to you quickly !
